vstools/utils/props.py

```diff
@@ -90,6 +93,16 @@ def get_prop(
 
 
 def merge_clip_props(*clips: vs.VideoNode, main_idx: int = 0) -> vs.VideoNode:
+    """
+    Merge frame properties from an unlimited number of clips.
+
+    The props of the main clips will not be overwritten, but more props can be added to it.
```

Comment:
The props of the main clip will be overridden as it's the main clip to where the other props will be added
The returned clip is the first clip in the sequence

Reply:
I'll leave rephrasing that better to you